test/commitments-specs.js

```diff
@@ -8,17 +8,23 @@ beforeEach(function syncDB () {
 })
 
 describe('commitments', () => {
-  context('GET /commitments/participant/:participant', () => {
-    it('should return empty if no commitments for participant with id=participant', async () => {
+  context('GET /commitments/participant/:fbid', () => {
+    it('should return error if no participant found with fbid=fbid', async () => {
       await koaRequest
-        .get('/commitments/participant/1')
+        .get('/commitments/participant/p0')
+        .expect(400)
```

Comment:
`204` to match the `GET /participants/:fbid` test in `test/participants-specs.js`